juniorguru/models/job.py

```diff
@@ -239,12 +240,14 @@ def get_employment_types_tags(types):
 
 
 class JobDropped(BaseModel):
+    id = IntegerField(null=True)
```

Comment:
Tomuhle nerozumím. Peewee by mělo ID přidávat automaticky a to, že není non null je mi podezřelé.

Reply:
V modelu musí být definované proto, aby se dalo volat save na modelu (myslím). `nullable` je buď kvůli nějaké zpětné kompatibilitě nebo kvůli tvoření nové tabulky po dropu. Podívám se na to.

Reply:
Tady je důvod, proč to bylo tak, jak to bylo: https://app.circleci.com/pipelines/github/honzajavorek/junior.guru/2391/workflows/db1ba334-638f-466c-bfb0-9edb9a1a9495/jobs/15534/parallel-runs/0/steps/0-104

Moc nevím, proč se to děje, když mi to pomůžeš rozluštit, budu rád.

Reply:
Ono to teda vlastně souvisí jenom trochu, protože se to týká Job (a ne JobDropped), ale mám pocit, že je to podobná oblast, tak ten komentář nechám tady.

Reply:
Tak už jsem se zorientoval - tenhle řádek je smazán.